fs_product_multi_image/views/fs_product_image.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="fs_product_image_form_view">
+        <field name="name">fs.product.image.form</field>
+        <field name="model">fs.product.image</field>
+        <field
+            name="inherit_id"
+            ref="fs_base_multi_image.fs_image_relation_mixin_form_view"
+        />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <group name="extra" position="inside">
+                <field name="tag_id" context="{'default_apply_on': 'product'}" />
+                <field
+                    name="available_attribute_value_ids"
+                    widget="many2many_tags"
+                    invisible="1"
+                />
+                <field
+                    name="attribute_value_ids"
+                    widget="many2many_tags"
+                    domain="[('id', 'in', available_attribute_value_ids)]"
+                    invisible="not available_attribute_value_ids"
+                />
+            </group>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="fs_product_image_kanban_view">
+        <field name="name">fs.product.image.kanban</field>
+        <field name="model">fs.product.image</field>
+        <field
+            name="inherit_id"
+            ref="fs_base_multi_image.fs_image_relation_mixin_kanban_view"
+        />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//kanban/field[@name='image']">
+                <field name="tag_id" />
+            </xpath>
+            <xpath expr="//div[hasclass('o_kanban_image')]" position="inside">
+                <div class="o_kanban_image me-1">
+                    <field
+                        name="image_medium"
+                        widget="image"
```

Comment:
The widget to use is 'fs_image' 
```suggestion
                        widget="fs_image"
```

With this change everything works properly on runboat.
![image](https://github.com/user-attachments/assets/12b85cd6-1704-4518-b3b1-b21c5181bd99)